src/helpers.ts

```diff
@@ -6,22 +6,22 @@ import {
     SMS_SUBSCRIPTION_CHANGED
 } from './events';
 
-export function isValidCallback(handler) {
+export function isValidCallback(handler: any) { // type will be checked in this method
```

Comment:
invariant throws an Error when the predicate fails right?

if so, how about using an assertion function: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions
```
function isValidCallback(x: any): asserts x is () => void {
    if (typeof x !== 'function') {
        throw 'not a valid fucntion'
    }
}
```

with invariant instead:
```
function isValidCallback(x: any): asserts x is () => void {
     invariant(
        typeof handler === 'function',
        'Must provide a valid callback'
    );
}
````

Reply:
Done, made handler of type `Function`